src/targets/pages.tmpl.js

```diff
@@ -0,0 +1,15 @@
+export const layout = "layouts/target.jsx";
+
+export default async function* ({ loadTargets, latestRelease }) {
+  // TODO:
+  return;
```

Comment:
Something missing here?

Reply:
yep, this is why the links to targets are broken. I dont generate any target pages during development for faster page generation :) Will fix :)